vertical-pod-autoscaler/README.md

```diff
@@ -54,6 +54,7 @@ The current default version is Vertical Pod Autoscaler 0.12.0
 
 | VPA version     | Kubernetes version |
 |-----------------|--------------------|
+| 0.13            | 1.25+              |
```

Comment:
If you're changing documentation as well - shouldn't the text above ("current default version") be also changed?
But maybe doc change should be a separate thing (e.g. part of Release Notes PR)?

Reply:
Done